packages/react-dom/src/__tests__/ReactDOM-test.js

```diff
@@ -311,14 +309,9 @@ describe('ReactDOM', () => {
     const actual = [];
 
     function click(node) {
-      const fakeNativeEvent = function() {};
-      fakeNativeEvent.target = node;
-      fakeNativeEvent.path = [node, container];
-      ReactTestUtils.simulateNativeEventOnNode(
-        TOP_CLICK,
-        node,
-        fakeNativeEvent,
-      );
+      ReactTestUtils.Simulate.click(node, {
+        path: [node, container],
+      });
```

Comment:
It would be event better if we completely dropped usage of Simulate or SimulateNative in these tests and just used DOM `dispatchEvent` like in some other tests. Note this only works when component container is mounted in the document. Again, exisiting tests with dispatchEvent show how to do it.

Reply:
Can we do this as a follow-up?

Reply:
Ok